### PR TITLE
Trigger donation popup

### PR DESCRIPTION
**Pull request by rishsingh07**
Trigger donation popup

### CODE REVIEW

**Comment by sagararyal** [comment]
Remove the console.log 
Besides that it looks good.

File: pages/[p].tsx [hunk 1]
```diff
@@ -44,17 +54,37 @@ export default function Donate({
       loadProject();
     }
   }, [router.query.p, currencyCode]);
-
+console.log(router.asPath,"router")
```
Comment (norbertschuler) [inline pages/[p].tsx]: please remove before merging

**Review by norbertschuler — CHANGES_REQUESTED** [review]
Tested https://planet-webapp-3qjq4hcx9-planetapp.vercel.app/community-based-restoration-at-el-silencio-natural-reserve#donate showing a popup and https://planet-webapp-3qjq4hcx9-planetapp.vercel.app/community-based-restoration-at-el-silencio-natural-reserve is not showing anything. Seems to work, but there is a error like
```
TypeError: Cannot read property 'includes' of undefined
    at _.fz.h (m=IZT63,ws9Tlc,p8L0ob,vfuNJf,PrPYRd,NpD4ec,Y2UGcc,SF3gsd,Ru0Pgb,hc6Ubd,ZyYHPb,Das5Le:441)
    at hz (m=IZT63,ws9Tlc,p8L0ob,vfuNJf,PrPYRd,NpD4ec,Y2UGcc,SF3gsd,Ru0Pgb,hc6Ubd,ZyYHPb,Das5Le:144)
    at _.jz.next (m=IZT63,ws9Tlc,p8L0ob,vfuNJf,PrPYRd,NpD4ec,Y2UGcc,SF3gsd,Ru0Pgb,hc6Ubd,ZyYHPb,Das5Le:145)
    at m=IZT63,ws9Tlc,p8L0ob,vfuNJf,PrPYRd,NpD4ec,Y2UGcc,SF3gsd,Ru0Pgb,hc6Ubd,ZyYHPb,Das5Le:145
    at new Promise (<anonymous>)
    at kz (m=IZT63,ws9Tlc,p8L0ob,vfuNJf,PrPYRd,NpD4ec,Y2UGcc,SF3gsd,Ru0Pgb,hc6Ubd,ZyYHPb,Das5Le:145)
    at Object._.lz (m=IZT63,ws9Tlc,p8L0ob,vfuNJf,PrPYRd,NpD4ec,Y2UGcc,SF3gsd,Ru0Pgb,hc6Ubd,ZyYHPb,Das5Le:146)
    at KU (m=IZT63,ws9Tlc,p8L0ob,vfuNJf,PrPYRd,NpD4ec,Y2UGcc,SF3gsd,Ru0Pgb,hc6Ubd,ZyYHPb,Das5Le:440)
    at _.fz.h (m=IZT63,ws9Tlc,p8L0ob,vfuNJf,PrPYRd,NpD4ec,Y2UGcc,SF3gsd,Ru0Pgb,hc6Ubd,ZyYHPb,Das5Le:440)
    at hz (m=IZT63,ws9Tlc,p8L0ob,vfuNJf,PrPYRd,NpD4ec,Y2UGcc,SF3gsd,Ru0Pgb,hc6Ubd,ZyYHPb,Das5Le:144)
```

Unfortunately with webpack5 source map generation on Vercel seems to have a problem, so we do not get a clear stacktrace.

File: pages/[p].tsx (hunk 1 above)
Comment (rishsingh07) [reply pages/[p].tsx]: already removed

**Review by harshvitra — APPROVED** [review]

**Pull request merged**